### ISSUE TITLE
ModSpec: Conformance class incorrectly specifies conformance tests as "Provisions"

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iso#812]
"Provisions" here should be "Conformance tests". Document: https://github.com/metanorma/iso-19115-3/

<img width="854" alt="Screen Shot 2022-10-05 at 4 51 25 PM" src="https://user-images.githubusercontent.com/11865/194020756-3717f5fe-30cb-4d5b-acde-fc45c809b386.png">

**Comment by opoudjis** [comment]
Liquid template is not populating the conformance tests at all.

YAML is converting URIs into link elements:

`conformance-test:: https://standards.isotc211.org/19115/-1/1/conf/metadata-core/basic`

is being processed as:

````
<dt>conformance-test</dt>
<dd><p><link target="https://standards.isotc211.org/19115/-1/1/conf/metadata-core/basic"/></p>\n</dd>
````

 Gem is not expecting that, but rather is expecting literal text.

**Comment by ronaldtse** [comment]
@opoudjis I'm not seeing this fixed. I'm pulling from Git gems. What am I supposed to try?

**Comment by ronaldtse** [comment]
Actually this is fixed in #813 . Thanks!
